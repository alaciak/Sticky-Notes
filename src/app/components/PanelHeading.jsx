import React from 'react';
import AddNewButton from '../containers/AddNewButton.jsx';
import { Provider } from 'react-redux';
import store from '../Store.jsx';

export class PanelHeading extends React.Component {

  render() {
    return (
      <header>
        <nav className='row'>
          <div className='col-md-12 page-header'>
            <h1>Sticky Notes Board</h1>
          </div>
        </nav>
        <div className='row'>
          <div className='col-md-6'>
            <Provider store={ store }>
              <AddNewButton />
            </Provider>
          </div>
        </div>
    </header>
    );
  }
}
