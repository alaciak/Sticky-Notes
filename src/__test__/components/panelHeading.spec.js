import React from 'react';
import { PanelHeading } from '../../app/components/PanelHeading.jsx';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('PanelHeading', () => {

  it('should render PanelHeading component', () => {

    const wrapper = shallow(<PanelHeading />);

    expect(toJson(wrapper)).toMatchSnapshot();

  });

});
