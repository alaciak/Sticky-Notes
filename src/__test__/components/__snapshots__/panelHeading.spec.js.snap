// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PanelHeading should render PanelHeading component 1`] = `
<header>
  <nav
    className="row"
  >
    <div
      className="col-md-12 page-header"
    >
      <h1>
        Sticky Notes Board
      </h1>
    </div>
  </nav>
  <div
    className="row"
  >
    <div
      className="col-md-6"
    >
      <Provider
        store={
          Object {
            "dispatch": [Function],
            "getState": [Function],
            "replaceReducer": [Function],
            "subscribe": [Function],
          }
        }
      >
        <Connect(AddNewButton) />
      </Provider>
    </div>
  </div>
</header>
`;
